tests/test_modeling.py

```diff
@@ -174,6 +174,13 @@ def test_save_model(self):
             self.assertTrue(NEURON_FILE_NAME in folder_contents)
             self.assertTrue(CONFIG_NAME in folder_contents)
 
+    def test_save_load_model(self):
+        # a config.json can change when converted and saved, so check loadability is maintained
+        with tempfile.TemporaryDirectory() as tmpdirname:
```

Comment:
Here I am not sure fully understood the test.

Are you worried that there might be a problem with re-loading the config if we save it once again? (for exemple in the case of this PR, users previously saved their models can run `model.save_pretrained()` to update their config with compiler version info)

If so, I think it is good to think of it but not really mandatory to add this test, as `NeuronBaseModel` is designed to be quite tolerant when having extra or lacking of not mandatory information in the config file.

Reply:
I used a test like this to ensure that the new config.json fields were being correctly processed when the NeuronBaseModel was loaded.  It did catch a bug where I had ">=" on the version check, when I should have just had ">".  

You're right about the tolerances of NeuronBaseModel.  I think this test served its purpose and is unlikely to catch anything in the future.  I'll remove it.